johnsonandjohnson/Bodiless-JS
# chore: fix export type warnings

## Changes
* fixed type warnings that can be seen in the terminal and browser console
* updated typescript, babel, eslint packages
* disabled arrow-parens, max-classes-per-file, react/jsx-props-no-spreading eslint rules which are enabled after eslint packages upgrade

## Test Instructions
* open test site in edit mode in browser
* view browser console logs

**actual:** type errors are present
**expected:** no type errors

## References
* https://devblogs.microsoft.com/typescript/announcing-typescript-3-8-beta/#type-only-imports-exports
* https://github.com/babel/babel/issues/10981
